Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/resources/css/pf-override.css

```diff
@@ -849,6 +849,82 @@ div[id*="cmOrgForm"] table.ui-panelgrid-domain {
    background-image:url("../resources/images/ui-icons_ffffff_256x240.png");
 }
 
+.form-button .ui-button {
+   margin:0px 0px 0px 10px;
+   padding:6px 12px;
+   color:#ffffff;
+   background:#428bca;
+   border:1px solid #357ebd;
+   text-shadow:none;
+   box-shadow:none;
+   white-space:nowrap;
+}
+.form-button .ui-button:first-child {
+   margin-left:0px;
+}
+.form-button .ui-button.ui-state-hover {
+    background:#3276b1;
+   border:1px solid #285e8e;
+   box-shadow:none;
+}
+.form-button .ui-button-text-only .ui-button-text {
+   padding:5px 10px;
+}
+.form-button .ui-button-text-icon-left .ui-button-text {
+   padding: .3em .3em .3em 1.3em;
+}
+.form-button .ui-button.btn-default {
+   background-color: #ffffff !important;
+   color: #333333 !important;
+   border-color: #cccccc !important;
+}
+.form-button .ui-button.btn-default:hover {
+   background-color: #eeeeee !important;
+}
+.form-button .ui-button.btn-primary .ui-icon {
+   background-image:url("../resources/images/ui-icons_ffffff_256x240.png");
```

Comment:
Can we use a glphycon instead of a png

Reply:
currently the glyphicon is not a image; it doesn't play well with primefaces; ignore this until we can fixed the pf-override to support glyphicon in the buttons